core/modules/file/js/file.js

```diff
@@ -37,10 +37,10 @@ Backdrop.behaviors.fileButtons = {
  */
 Backdrop.behaviors.filePreviewLinks = {
   attach: function (context) {
-    $('div.form-managed-file .file a, .file-widget .file a', context).once('file-preview-link').on('click', Backdrop.file.openInNewWindow);
+    $('.file-preview-link', context).once('file-preview-link').on('click', Backdrop.file.openInNewWindow);
```

Comment:
@stpaultim I updated your PR so it uses `on()` and `off()` rather than `bind()` and `unbind()` -- the latter are deprecated.